.changeset/few-tomatoes-march.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': major
+---
+
+Button component sizes have been updated with the new values. The default **_mega_** size value has been changed to **_giga_**.
```

Comment:
```suggestion
Increased the Button sizes to match other form components and renamed the default size from `mega` to `giga`.
```

Is there a codemod for this? If so, please mention it in the changeset (e.g. "🤖 _button-default-size_".